src/collections/Menu/Menu.d.ts

```diff
@@ -81,7 +81,7 @@ export interface MenuProps {
   stackable?: boolean;
 
   /** A menu can be formatted to show tabs of information. */
-  tabular?: boolean | 'right';
+  tabular?: boolean | 'right' | 'left';
```

Comment:
Can you clarify necessity of `left` value? There [is no special](https://github.com/Semantic-Org/UI-Menu/blob/master/menu.css#L683) CSS for it, so it's `left` by default.